dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -280,8 +281,10 @@ private void createFolder() {
     }
 
     private void initProgressAndPathIndicators() {
-        final Stream<FileTreeModel> rootItems = fileTreeTableProvider.getRoot(fileRootItem);
-        localPathIndicator.setText(StringConstants.ROOT_LOCATION);
+        final String userHome = System.getProperty(StringConstants.SETTING_FILE_PATH);
```

Comment:
Can we rename the `SETTING_FILE_PATH` constant to something that more clearly specifies that it's the user's home directory.

Reply:
There we go